src/coreclr/gc/gcpriv.h

```diff
@@ -142,6 +142,14 @@ inline void FATAL_GC_ERROR()
 #define USE_REGIONS
 #endif //HOST_64BIT && BUILD_AS_STANDALONE
 
+#define SPINLOCK_HISTORY
+#define RECORD_LOH_STATE
```

Comment:
```suggestion
// #define SPINLOCK_HISTORY
// #define RECORD_LOH_STATE
```

Reply:
you wouldnt want to define these unconditionally - at least not in release builds

Reply:
Will do.